rotkehlchen/db/filtering.py

```diff
@@ -568,6 +569,30 @@ def prepare(self) -> tuple[list[str], list[T]]:
         )
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBMultiStringAsFloatRangeFilter(DBFilter):
+    """Filters a column having a string value that represents a decimal, out of a selection of
+    decimal values. It also allows a tolerance for the comparison."""
+
+    column: str
+    values: list[FValWithTolerance]
+
+    def prepare(self) -> tuple[list[str], list[str]]:
+        conditions = []
+        parameters = []
+
+        for val in self.values:
+            value = val[0]
+            tolerance = val[1] * FVal(0.5)
```

Comment:
Because the range is created by adding half the tolerance to the given value and removing half the tolerance from it so that the total tolerance is the given tolerance number. 
`parameters.extend([str(value - tolerance), str(value + tolerance)])`

Reply:
I see. This makes sense. Can you maybe add a tiny comment next to it explaining this. In case others reading the code have the same question?